daemon/Cargo.toml

```diff
@@ -45,7 +45,7 @@ tracing = { version = "0.1" }
 tracing-subscriber = { version = "0.2", default-features = false, features = ["fmt", "ansi", "env-filter", "chrono", "tracing-log", "json"] }
 uuid = { version = "0.8", features = ["serde", "v4"] }
 x25519-dalek = { version = "1.1" }
-xtra = { version = "0.6", features = ["with-tokio-1"] }
+xtra = { version = "0.6" }
```

Comment:
I believe this patch by itself will fail to compile yes? It should probably come after you've migrated all usages of `spawn_global`.

Reply:
It didn't fail to compile. We were not using `spawn_global` before this changeset. I moved it after as it didn't matter.